About/history.html

```diff
@@ -143,30 +142,31 @@
       title="DARPA HACMS: seL4 protects unmanned helicopter against cyber-attacks"
       description='
       <p>
-        The impact of seL4&apos;s verification was immediately significant in the
-	<a href="../Research/">Research</a> area, with high-impact
+        In the research area, the impact of seL4&apos;s verification is immediately
+        apparent with high-impact
 	<a href="../Research/publications.html">publications</a> spanning the
-	disciplines of Systems, Formal Methods, and Software Engineering. Its lasting real-world
-	significance was marked by the
+	disciplines of Systems, Formal Methods, and Software Engineering. For
+  lasting real-word significance, it is the
 	DARPA-funded <a href="http://loonwerks.com/projects/hacms.html">HACMS
-	program</a>, where seL4 was demonstrated and embedded in a range of autonomous vehicles,
-	ranging from trucks, a land robot, and a quadcopter to Boeing&apos;s
+	program</a> that demonstrates and embeds seL4 in a range of autonomous vehicles,
+	ranging from trucks, a land robot, and a quadcopter, to Boeing&apos;s
```

Comment:
I would like to suggest:

While the research impact of seL4&apos;s verification is immediately apparent, with high-impact <a href="../Research/publications.html">publications</a> spanning the
	disciplines of Systems, Formal Methods, and Software Engineering, its lasting real-word significance is first demonstrated in the DARPA-funded HACMS program. In that program, seL4 is embedded [...]